src/doc/reference/reducers.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Reducers
+author: John F. Carr
+---
+
+# Reducers
```

Comment:
I'm not sure how everything will look once this page is merged with style fixes in other PRs.  But right now I think this line can be removed, since the title "Reducers" already show up elsewhere on the page.